doc/Addons.md

```diff
@@ -321,184 +321,299 @@ Complete list of hook callbacks
 
 Here is a complete list of all hook callbacks with file locations (as of 01-Apr-2018). Please see the source for details of any hooks not documented above.
 
-index.php:	Addon::callHooks('init_1');
-index.php:	Addon::callHooks('app_menu', $arr);
-index.php:	Addon::callHooks('page_content_top', $a->page['content']);
-index.php:	Addon::callHooks($a->module.'_mod_init', $placeholder);
-index.php:	Addon::callHooks($a->module.'_mod_init', $placeholder);
-index.php:	Addon::callHooks($a->module.'_mod_post', $_POST);
-index.php:	Addon::callHooks($a->module.'_mod_afterpost', $placeholder);
-index.php:	Addon::callHooks($a->module.'_mod_content', $arr);
-index.php:	Addon::callHooks($a->module.'_mod_aftercontent', $arr);
-index.php:	Addon::callHooks('page_end', $a->page['content']);
+### index.php
+
+    Addon::callHooks('init_1');
+    Addon::callHooks('app_menu', $arr);
+    Addon::callHooks('page_content_top', $a->page['content']);
+    Addon::callHooks($a->module.'_mod_init', $placeholder);
+    Addon::callHooks($a->module.'_mod_init', $placeholder);
+    Addon::callHooks($a->module.'_mod_post', $_POST);
+    Addon::callHooks($a->module.'_mod_afterpost', $placeholder);
+    Addon::callHooks($a->module.'_mod_content', $arr);
+    Addon::callHooks($a->module.'_mod_aftercontent', $arr);
+    Addon::callHooks('page_end', $a->page['content']);
+    
+### include/api.php
+	Addon::callHooks('logged_in', $a->user);
```

Comment:
Should these be indented as well and prefaced by an empty line.

Reply:
Fixed